src/main/resources/entity-types/inventory/simple_instance_type.json5

```diff
@@ -0,0 +1,147 @@
+{
+  id: 'af44e2e0-12e0-4eec-b80d-49feb33a866c',
+  name: 'simple_instance_type',
+  private: true,
+  sources: [
+    {
+      type: 'db',
+      alias: 'instance_type',
+      target: 'src_inventory_instance_type',
+    },
+  ],
+  requiredPermissions: [
+    'inventory-storage.instance-types.collection.get',
+    'inventory-storage.instance-types.item.get'
+  ],
+  columns: [
+    {
+      name: 'id',
+      sourceAlias: 'instance_type',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      isIdColumn: true,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'id'",
+      source: {
+        entityTypeId: 'af44e2e0-12e0-4eec-b80d-49feb33a866c',
+        columnName: 'id',
+      },
+      valueSourceApi: {
+        path: 'instance-types',
+        valueJsonPath: '$.instanceTypes.*.id',
+        labelJsonPath: '$.instanceTypes.*.id',
+      }
+    },
+    {
+      name: 'code',
+      sourceAlias: 'instance_type',
+      dataType: {
+        dataType: 'stringType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: true,
+      valueGetter: ":sourceAlias.jsonb->>'code'",
+      filterValueGetter: "lower(\
+        ${tenant_id}_mod_inventory_storage.f_unaccent (:sourceAlias.jsonb->>'code'::text)\
+      )",
+      valueFunction: 'lower(${tenant_id}_mod_inventory_storage.f_unaccent (:value))',
+      source: {
+        entityTypeId: 'af44e2e0-12e0-4eec-b80d-49feb33a866c',
+        columnName: 'code',
+      },
+      valueSourceApi: {
+        path: 'instance-types',
+        valueJsonPath: '$.instanceTypes.*.code',
+        labelJsonPath: '$.instanceTypes.*.code',
+      }
```

Comment:
These fields with `valueSourceApi`s all have backing indexes, so there's no need to redirect them to the `id` field with `idColumnName`